src/components/BulkActionsToolbar/__tests__/BulkActionsToolbar.spec.tsx

```diff
@@ -88,14 +92,15 @@ describe('BulkActionsToolbar component', () => {
                 actions={actions}
             />
         );
-        wrapper.find('.BulkActionsToolbar__toggleButton').simulate('click');
+        await userEvent.click(screen.getByRole('button', { name: 'Select All' }));
```

Comment:
I also saw it in the other [PR](https://github.com/textkernel/oneui/pull/949#discussion_r1178133030). It is not clear to me when we need/should initialise a user with `userEvent.setup()`. Can you look into it, and provide some clarity around it? Thanks

Reply:
I've checked some articles and only find that it recommened to use https://testing-library.com/docs/user-event/setup/